Fixes #21003 - Improve table for ansible roles

bastilian opened the pull request:
Before:
![gnome-shell-screenshot-7sit9y](https://user-images.githubusercontent.com/7757/32714905-361aed38-c850-11e7-9694-7366b5055aa3.png)

After:
![gnome-shell-screenshot-p3za9y](https://user-images.githubusercontent.com/7757/32714914-40005b44-c850-11e7-9144-de4ddaceee62.png)

dLobatog commented on app/controllers/ansible_roles_controller.rb:
```diff
@@ -5,10 +5,11 @@ class AnsibleRolesController < ::ApplicationController
   def index
     @ansible_roles = resource_base.search_for(params[:search],
-                                              :order => params[:order]).
+                                              :order => ()).
```
Shouldn't this still be `params[:order]` in order to use `default_order`?

bastilian replied:
Yes, it should. fixed.

ares commented:
Could we also have columns containing number/date more thin? values there have fixed size, the name though can be quite long. Could you please add ordering to the rest of columns? At lesat imported_at if counts are too complicated.

bastilian commented:
@ares & @dLobatog:

* Adjusted the width of all colums
* Ordering by "Imported at" is now possible
* Added scoped_search for ansible_roles to `Host` to allow searching for `role =`
* Hosts count is linked to roles search of hosts

![gnome-shell-screenshot-7mzo9y](https://user-images.githubusercontent.com/7757/32725575-e35590a0-c875-11e7-92e6-5d7f725c97f8.png)

dLobatog commented:
@bastilian I think https://github.com/theforeman/foreman_ansible/commit/62c71aa1ecbff4757584d8800d75b34ec0f4e943 was causing one test to fail. If so, we should see http://ci.theforeman.org/job/test_plugin_foreman_ansible_master/ green again (at least I do locally). If it was something else, I'll merge this & work on the error after.

dLobatog commented:
Thanks @bastilian !
